book/src/background/fields.md

```diff
@@ -37,32 +37,33 @@ Important notes:
   versa. We could also use whether or not an element is larger than $(p - 1) / 2$ to give
   it a "sign."
 
-## Inverses and groups
+Finite fields will be useful later for constructing [polynomials](polynomials.md) and
+[elliptic curves](curves.md). Elliptic curves are examples of groups, which we discuss
+next.
 
-Any non-zero element $a \in \mathbb{F}_p$ has a _multiplicative inverse_ $b = a^{-1}$,
-which is the _unique_ element $b$ such that $ab = 1$.
+## Groups
 
-A quick way of obtaining the inverse is $a^{-1} = a^{p - 2}$. The reason for this stems
-from [Fermat's little theorem][fermat-little], which states that $a^p = a \pmod p$ for any
-integer $a$. If $a$ is nonzero, we can divide by $a$ twice to get $a^{p-2} = a^{-1}.$
+Groups are simpler and more limited than fields; they have only one binary operator $\cdot$
+and fewer axioms. They also have an identity, which we'll denote as $1$.
```

Comment:
```suggestion
and fewer axioms. They also have an identity, which we'll denote as $\mathcal{O}$.
```
(since 1 collides with the isomorphism-to-integers discussion later, and this is consistent with that section)

Reply:
No this would be more confusing. The standard notation for the additive and multiplicative identities of a field are 0 and 1 respectively. We only use \mathcal{O} for the identity of a group that is written additively.

I don't agree that it collides with the isomorphism-to-integers discussion. The sides of the isomorphism are different structures and could each be written with any notation.

Reply:
Also, although we *can* choose any notation, I think we *should* avoid having \mathcal{O} in an additively written group and \mathcal{O} in the scalar field that are *not* isomorphic (because the group identity corresponds to 0 in the scalar field, not 1).